test/type/self.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 5
```

Comment:
You need to test the old behavior in test/Compatibility/self.swift or similar, too.

Reply:
Will do!